docker-compose.yml

```diff
@@ -61,7 +60,7 @@ services:
       - ./frontend/static:/usr/src/robosats/static
 
   clean-orders:
-    image: backend
+    build: .
```

Comment:
I include this not as change but more like a question, docker-compose was not able to find any `backend` since it's not defined anywhere. This was the only way I manage to make it run, am I missing something?